src/settings.ts

```diff
@@ -192,6 +217,46 @@ export class LabelsSettings extends FormattingSettingsSimpleCard {
         value: {value: "white"},
     });
 
+    public fontControl: formattingSettings.FontControl = new formattingSettings.FontControl({
+        name: "fontControl",
+        displayNameKey: "Visual_Font_Control",
+        fontFamily: new formattingSettings.FontPicker({
+            name: "fontFamily",
+            displayNameKey: "Visual_Font",
+            value: "Arial"
```

Comment:
Usually we do use "Segoe UI, wf_segoe-ui_normal, helvetica, arial, sans-serif" as default font.

Reply:
Same on line 118

Reply:
changed to "Segoe UI, wf_segoe-ui_normal, helvetica, arial, sans-serif"